src/neo/Hardfork.cs

```diff
@@ -12,6 +12,7 @@ namespace Neo
 {
     public enum Hardfork : byte
     {
+        HF_2693_FixGetRandom,
```

Comment:
Can we _not_ introduce another hardfork and have one hardfork per release? It's going to be a configuration nightmare soon this way.

Reply:
How to name it?

Reply:
Naming is hard.

DevonRex, Chihuahua, Orizaba, Agapovka --- all work fine for me and can start some interesting thematic sequence. But I guess we can have hundreds of other proposals here soon.

Reply:
Name of mountains, rivers, cities, animals,,,,,,,

Reply:
Or just the version number? `HF_3_3_0`?

Reply:
This works of course, but it's somewhat boring. Codenaming these things is preferable to me. To keep track of which hf comes after another alphabet could be used, something like Annapurna -> Belukha -> Chimborazo sequence.

Anyway, the main thing is to have one hf per release.

Reply:
Aspidochelone -> Basilisk -> Cockatrice?

Reply:
![dejavu](https://user-images.githubusercontent.com/22092804/169763019-c0765b1d-1aa6-4a35-9430-a8119bbcd0cd.jpeg)